### DISCUSSION TITLE
Request to be a member of the OceanModeling Org

### DISCUSSION THREAD

**Discussion by saeed-moghimi-noaa** [oceanmodeling/Management#17] [General]
Please add a note and tag me @saeed-moghimi-noaa.

**Comment by saeed-moghimi-noaa** [comment]
All, 

Just to inform you about our up coming bi-monthly tag up. Let me know if you want to add a topic to be discussed at the meeting.

Possible agenda items:
- UFS-Coastal updates
- Searvey Updates
- Up coming DMAC workshop updates
- !?

Best,
-Saeed

Ocean Modeling Collaboration Bi-monthly Meeting
Thursday, April 4 · 09:00 – 09:50
Time zone: America/New_York
Google Meet joining info
Video call link: https://meet.google.com/eyy-zqbd-ppr
Or dial: ‪(US) +1 208-715-5331‬ PIN: ‪260 201 088‬#
More phone numbers: https://tel.meet/eyy-zqbd-ppr?pin=9418066551126

**Comment by tomsail** [comment]
Please @saeed-moghimi-noaa can you add Sebastien Bourban @sebourban and Christophe Coulet @ccoulet for the next meeting ?

**Reply by saeed-moghimi-noaa** [reply]
Hi @tomsail 

Would you please add their emails?

Thanks,
-Saeed

**Reply by tomsail** [reply]
sebastien.bourban@edf.fr 
c.coulet@brgm.fr

**Reply by ccoulet** [reply]
Thanks for the invitation to participate @saeed-moghimi-noaa and @tomsail

**Comment by saeed-moghimi-noaa** [comment]
Done. Next meeting open to ALL:
### Ocean Modeling Collaboration Bi-monthly Meeting
Thursday, June 6 · 09:00 – 10:30
Time zone: America/New_York
Google Meet joining info
Video call link: https://meet.google.com/eyy-zqbd-ppr
Or dial: ‪(US) +1 208-715-5331‬ PIN: ‪260 201 088‬#
More phone numbers: https://tel.meet/eyy-zqbd-ppr?pin=9418066551126

**Comment by sebourban** [comment]
Dear @saeed-moghimi-noaa - can you please also add two other colleagues from EDF : 

- At EDF UK : Jacob BURROWS <Jacob.Burrows@edfenergy.com> 
- At EDF R&D China : MA Xinran <xinran.ma@edf.fr>

Thank you,

Looking forward to participating,
Best regards,
Sébastien (at EDF R&D France).

**Reply by saeed-moghimi-noaa** [reply]
@sebourban @tomsail 
Welcome to the team. I included all. I think you will receive an email.  Feel free to attend or add others as you see fit. Thanks

**Comment by AliS-Noaa** [comment]
Hello @tomsail : can you please add mcc276@cornell.edu to the meetings. Cheers,

**Comment by veenstrajelmer** [comment]
@saeed-moghimi-noaa please add me to the list (jelmer.veenstra@deltares.nl)
